tests/autoload.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+// phpcs:ignoreFile
```

Comment:
ignoreFile should not be used in bootstrap:
https://github.com/wearesho-team/bobra-authorization/commit/eb8de8923c621916ce91f7aa435890a0b1ecc180